src/rasenmaeher_api/web/api/people/schema.py

```diff
@@ -0,0 +1,10 @@
+"""Schema for people."""
+from typing import List, Dict, Any
+
+from pydantic import BaseModel, Extra
+
+
+class PeopleListOut(BaseModel, extra=Extra.forbid):  # pylint: disable=too-few-public-methods
+    """People list out response schema"""
+
+    callsign_list: List[Dict[Any, Any]]
```

Comment:
Type this properly, make a Pydantic schema for the person object.

Also use Field(description="something") for all fields